_posts/0000-01-02-Zetthard.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
* hmm, **do** I need to put *space* after the star in the beginning of the line or I __don't__